docs/Installation.md

```diff
@@ -31,7 +31,7 @@ Docker are found there.
 The rest of this document is about doing a local installation of Zonemaster-CLI,
 not relevant for running Zonemaster-CLI on Docker.
 
-## Prerequisites
+## Prerequisites for CPAN installation
 
 Before installing Zonemaster::CLI, you should [install Zonemaster::Engine][
 Zonemaster::Engine installation].
```

Comment:
When you come to the headline you do not even know that there is an alternative with package. I suggest the following:

```
## Prerequisites for CPAN installation

Before installing Zonemaster::CLI from CPAN, you should [install Zonemaster::Engine][
Zonemaster::Engine installation], unless you are to install on Debian using pre-built
packages (see below).
```

Reply:
done